### PR TITLE
Extend CoreImage conformance 

### PR DESCRIPTION
**Pull request by Bellaposa**
I extended some CoreImage conformance

### CODE REVIEW

File: Sources/CustomDump/Conformances/CoreImage.swift [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+extension CGRect: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CGRect.(default, rawValue: \(self))"
+        }
+    }
+}
+
```
Comment (stephencelis) [inline Sources/CustomDump/Conformances/CoreImage.swift:107]:
```suggestion
```

I think we want to let `CGRect` render as it normally does. There's a test in CI that is failing for this PR as a result.

File: Sources/CustomDump/Conformances/CoreImage.swift [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+extension CIRAWFilterOption: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIRAWFilterOption.(default, rawValue: \(self.rawValue))"
+        }
+    }
+}
+
```
Comment (stephencelis) [inline Sources/CustomDump/Conformances/CoreImage.swift:116]: This, too, I think can lean on the default rendering behavior, right? It's also a type that's been deprecated.
```suggestion
```

File: Sources/CustomDump/Conformances/CoreImage.swift [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+extension CIKernel: CustomDumpStringConvertible {
+    public var customDumpDescription: String {
+        switch self {
+        default:
+            return "CIKernel.(default, rawValue: \(self))"
+        }
+    }
+}
+
```
Comment (stephencelis) [inline Sources/CustomDump/Conformances/CoreImage.swift:98]: Should any of these conformances be necessary? None are switching on enum-like values, so I think the default rendering should be ok, right?

**Review by stephencelis — COMMENTED** [review]
Thanks for taking the time to PR! Some of the conformances should be good to go, but I'm bit quite sure I understand all of them...

File: Sources/CustomDump/Conformances/CoreImage.swift (hunk 3 above)
Comment (Bellaposa) [reply Sources/CustomDump/Conformances/CoreImage.swift:98]: Yes, I agree!

File: Sources/CustomDump/Conformances/CoreImage.swift (hunk 2 above)
Comment (Bellaposa) [reply Sources/CustomDump/Conformances/CoreImage.swift:116]: That's ok!

**Comment by Bellaposa** [comment]
I updated by removing the unnecessary conformance.

**Review by stephencelis — APPROVED** [review]
Thanks!

**Pull request merged**